libs/blocks/marquee/marquee.css

```diff
@@ -148,6 +148,7 @@
 .marquee .image img {
   display: block;
   width: 100%;
+  min-height: 150px; /* fixed with breaking 05 */
```

Comment:
breaking 05?  What's that referring to?

Reply:
There's a breaking change that's going to be happening in Helix that will obviate the need to set this height.